tests/assistants/test_api.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import pytest
+
+from ragna import assistants
+from ragna._compat import anext
+from ragna.assistants._api import ApiAssistant
+from ragna.core import RagnaException
+from tests.utils import skip_on_windows
+
+API_ASSISTANTS = [
+    assistant
+    for assistant in assistants.__dict__.values()
+    if isinstance(assistant, type)
+    and issubclass(assistant, ApiAssistant)
+    and assistant is not ApiAssistant
+]
+
+
+@skip_on_windows
```

Comment:
I tried multiple things now, but got nothing to work. All data points to the test code being the problem rather than the code that is tested here. Let's ignore this for now. Maybe someone else has more insights later.